src/components/Authentication/OAuth.vue

```diff
@@ -0,0 +1,73 @@
+<template lang="pug">
+.oauth-widget(v-if="providers.length")
+  v-container
+    h4.ml-0 Or {{verb}} with OAuth
+    a.no-decorate(v-for="provider in providers",
+        :key="provider.id",
+        :href="provider.url")
+      v-btn(:dark="iconMap[provider.id].dark", :color="provider.id")
+        v-icon.aligned {{ $vuetify.icons[iconMap[provider.id].icon] }}
```

Comment:
Is the `aligned` class something from vuetify? It doesn't seem to have any effect for me locally.